src/Entity/Conference.php

```diff
@@ -11,10 +11,10 @@
 
 namespace App\Entity;
 
-use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\ArrayCollection;
 use Gedmo\Timestampable\Traits\TimestampableEntity;
```

Comment:
Why all theses diff? It's not alpha sorted anymore.

Did you run php-cs-fixer ?

Reply:
I did not run the CS fixer yet since it was still a WIP, as I didn't run phpstan or tests yet aswell.
Since there will still be plenty of modifications, I intend to do it once things are more stable.

However, if you feel this is making it uncomfortable to read, I can run at least the fixer